.markdownlint.yml

```diff
@@ -1,5 +1,9 @@
+{
```

Comment:
errr, the markdownlint config is described in YAML, hence the file extension name. I'm pretty confident JSON is supported to, but why make this change?

Reply:
Ah, I'll put it back into YAML, I was just making changes based on some docs that described how to disable certain linting tests and those docs happened to be in JSON